pkg/controllers/binding/common.go

```diff
@@ -48,26 +48,30 @@ func ensureWork(
 ) error {
 	bindingSpec := getBindingSpec(binding, scope)
 	targetClusters := mergeTargetClusters(bindingSpec.Clusters, bindingSpec.RequiredBy)
+	var err error
+	var errs []error
 
 	var jobCompletions []workv1alpha2.TargetCluster
-	var err error
-	if workload.GetKind() == util.JobKind {
+	if workload.GetKind() == util.JobKind && needReviseJobCompletions(bindingSpec.Replicas, bindingSpec.Placement) {
 		jobCompletions, err = divideReplicasByJobCompletions(workload, targetClusters)
 		if err != nil {
 			return err
 		}
 	}
 
-	var errs []error
 	for i := range targetClusters {
 		targetCluster := targetClusters[i]
 		clonedWorkload := workload.DeepCopy()
 
 		workNamespace := names.GenerateExecutionSpaceName(targetCluster.Name)
 
-		// If and only if the resource template has replicas, and the replica scheduling policy is divided,
-		// we need to revise replicas.
-		if needReviseReplicas(bindingSpec.Replicas, bindingSpec.Placement) {
+		// If and only if the resource template has replicas, we need to revise replicas.
+		// The replicas value in "Work" should be consistent with the scheduling result bindingSpec.Clusters,
+		// or bindingSpec.RequiredBy controlled by the dependencies-distributor, rather than bindingSpec.Replicas.
+		// This remains true even when the replica scheduling policy is duplicated.
+		// In certain scenarios, such as scheduling suspension or insufficient federation quota,
+		// bindingSpec.Replicas should not be directly synchronized to the member clusters.
```

Comment:
```suggestion
		// When syncing workloads to member clusters, the controller MUST strictly adhere to the scheduling results
		// specified in bindingSpec.Clusters for replica allocation, rather than using the replicas declared in the
		// workload's resource template.
		// This rule applies regardless of whether the workload distribution mode is "Divided" or "Duplicated".
		// Failing to do so could allow workloads to bypass the quota checks performed by the scheduler
		// (especially during scale-up operations) or skip queue validation when scheduling is suspended.
```
How about this?